keps/sig-multicluster/1645-multi-cluster-services-api/README.md

```diff
@@ -715,30 +717,25 @@ to describe a selector based policy that applies to a multi-cluster service.
 
 _Optional, but recommended._
 
-MCS aims to align with the existing [service DNS
+The full specification for Multicluster Service DNS is in this KEP's [specification.md](specification.md). MCS aims to align with the existing [service DNS
```

Comment:
minor - most MD docs I see linewrap at 80 :)

Reply:
rewrapped everything in 270408a, 8144f35 and unwrapped TOC in 7d8d00d :) 

this message brought to you by the [Rewrap VSCode extension](https://marketplace.visualstudio.com/items?itemName=stkb.rewrap)